openlogin/authorized-apps/index.test.ts

```diff
@@ -0,0 +1,157 @@
+import { expect, Page } from "@playwright/test";
+import { test } from "./index.lib";
+import { useAutoCancel2FASetup } from "../../utils";
+import { useAutoCancelShareTransfer } from "../../utils/index";
+import Mailosaur from "mailosaur";
+import { Link } from "mailosaur/lib/models";
+
+const mailosaur = new Mailosaur(process.env.MAILOSAUR_API_KEY || "");
+
+const testEmail = `hello+apps+${Date.now()}@${
+  process.env.MAILOSAUR_SERVER_DOMAIN
```

Comment:
It would be useful to extract out a mailosaur utils, which can have utility functions like findLink, generateRandomTestEmail. @sidharthWeb3auth

Reply:
done